pkgs/by-name/lx/lxgw-fusionkai/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lxgw-fusionkai";
+  version = "v24.134";
+
+  src = fetchFromGitHub {
+    owner = "lxgw";
+    repo = "FusionKai";
+    rev = version;
+    hash = "sha256-pEISoFEsv8SJOGa2ud/nV1yvl8T9kakfKENu3mfYA5A=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    mv *.ttf $out/share/fonts/truetype
```

Comment:
We can use the `install` command, as in `pkgs/data/fonts/cascadia-code/default.nix`

Reply:
Is this a documented requirement to prefer to use "install"?